extensions/wikia/AchievementsII/services/AchImageUploadService.class.php

```diff
@@ -145,74 +146,74 @@ public static function uploadBadge($destinationFileName, $badgeLevel) {
 		}
 
 		// resize and crop uploaded image / center inside a badge
-		$tmp = imagecreatetruecolor(128, 128);
+		$tmp = imagecreatetruecolor( 128, 128 );
 
 		// transparent background
-		imagesavealpha($tmp, true);
-		imagealphablending($tmp, false);
+		imagesavealpha( $tmp, true );
+		imagealphablending( $tmp, false );
 
-		$bgColor = imagecolorallocatealpha($tmp, 0, 0, 0, 127);
-		imagefilledrectangle($tmp, 0, 0, 127, 127, $bgColor);
+		$bgColor = imagecolorallocatealpha( $tmp, 0, 0, 0, 127 );
+		imagefilledrectangle( $tmp, 0, 0, 127, 127, $bgColor );
 
 		// calculate crop / resize
-		$dstX = max(0, (128 - $newWidth) >> 1);
-		$dstY = max(0, (128 - $newHeight) >> 1);
+		$dstX = max( 0, ( 128 - $newWidth ) >> 1 );
+		$dstY = max( 0, ( 128 - $newHeight ) >> 1 );
 
-		$srcX = max(0, ($badgeWidth - 128 * $ratio) >> 1);
-		$srcY = max(0, ($badgeHeight - 128 * $ratio) >> 1);
+		$srcX = max( 0, ( $badgeWidth - 128 * $ratio ) >> 1 );
+		$srcY = max( 0, ( $badgeHeight - 128 * $ratio ) >> 1 );
 
-		imagecopyresampled($tmp, $badgeImage, $dstX, $dstY, $srcX, $srcY, $newWidth, $newHeight, $badgeWidth, $badgeHeight);
+		imagecopyresampled( $tmp, $badgeImage, $dstX, $dstY, $srcX, $srcY, $newWidth, $newHeight, $badgeWidth, $badgeHeight );
 
 		// use resized badge
-		imagedestroy($badgeImage);
+		imagedestroy( $badgeImage );
 		$badgeImage = $tmp;
 
-		if ( !empty( $wgBadgeNoFrame ) ) {
+		if( !empty( $wgBadgeNoFrame ) ) {
 			// save badge as is
-			imagepng($badgeImage, $badgeFile, 9, PNG_ALL_FILTERS);
+			imagepng( $badgeImage, $badgeFile, 9, PNG_ALL_FILTERS );
 		} else {
 			//load images
-			$badgesFragments = dirname(__FILE__) . '/../images/badges/fragments';
+			$badgesFragments = dirname( __FILE__ ) . '/../images/badges/fragments';
 
 			$badgeBottom = "{$badgesFragments}/{$badgeLevel}-bottom-128.png";
 			$badgeTop = "{$badgesFragments}/{$badgeLevel}-top-128.png";
 
 			$img = array(
-				'bottom' => imagecreatefrompng($badgeBottom),
-				'top' => imagecreatefrompng($badgeTop),
+				'bottom' => imagecreatefrompng( $badgeBottom ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "bottom" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)